tests/rest_api/test_projects.py

```diff
@@ -0,0 +1,214 @@
+    @pytest.mark.parametrize('privilege', ('admin', 'business', 'user'))
+    def test_is_user_can_create_project(self, users, find_users, privilege, **kwargs):
+        privileged_users = find_users(privilege=privilege)
+        assert len(privileged_users)
+
+        username = privileged_users[0]['username']
+        spec = {
+            'name': f'test {username} tries to create a project'
+        }
+        self._test_create_project_201(username, spec, **kwargs)
+
+    # User with user privilege cannot have more than three projects (POST /projects/).
+    def test_if_user_cannot_have_more_than_3_projects(self, users, projects, find_users,  **kwargs):
+        user_users = find_users(privilege='user')
+        assert len(user_users)
+
+        user = user_users[0]
+        user_projects = 0
+        for p in projects:
+            if user['id'] == p['owner']['id']:
+                user_projects += 1
+
+        assert user_projects <= 3
+
+        for i in range(1, 4 - user_projects):
+            spec = {
+                'name': f'test: {user["username"]} tries to create a project number {user_projects + i}'
+            }
+            self._test_create_project_201(user['username'], spec, **kwargs)
+
+        spec = {
+                'name': f'test {user["username"]} tries to create more than 3 projects'
+            }
+        self._test_create_project_403(user['username'], spec, **kwargs)
+
+    # User with admin or business role can have more than three projects (POST /projects).
+    @pytest.mark.parametrize('privilege', ('admin', 'business'))
+    def test_if_user_can_have_more_than_3_projects(self, users, projects, find_users, privilege, **kwargs):
+        privileged_users = find_users(privilege=privilege)
+        assert len(privileged_users)
+
+        user = privileged_users[0]
+
+        for i in range(1, 5):
+            spec = {
+                'name': f'test: {user["username"]} with privilege {privilege} tries to create a project number {i}'
+            }
+            self._test_create_project_201(user['username'], spec, **kwargs)
+
+    # [organization]  Member of organization that has role worker cannot create a project
+    # (POST /projects/)
+    @pytest.mark.parametrize('org, project_id', [({'id': 2, 'slug': 'org2'}, 2)])
+    def test_if_org_worker_cannot_crate_project(self, users, find_users, org, project_id, **kwargs):
+        workers = find_users(org=org['id'], role='worker')
+        assert len(workers)
+
+        worker = workers[0]
+
+        spec = {
+            'name': f'test: worker {worker["username"]} creating a project for his organization',
+            'organization_id': org['id'],
+            'org': org['slug']
+        }
+        self._test_create_project_403(worker['username'], spec, org_id=org['id'])
+
+    # [organization]  Member of organization that has role supervisor, maintainer or owner can create a project
+    # (POST /projects/)
```

Comment:
Please, leave a comment for tests in case when it's really necessary. For example, in this case I think we don't need to mention `(POST /projects)` because this `TestPostProject` test class. And developers can easy understand what this test really do from the name of test function.